core/extension/gdextension_interface.h

```diff
@@ -590,7 +590,7 @@ typedef GDExtensionInterfaceFunctionPtr (*GDExtensionInterfaceGetProcAddress)(co
  *
  * For example:
  *
- *   GDExtensionInterfaceGetGodotVersion *get_godot_version = (GDExtensionInterfaceGetGodotVersion)p_get_proc_address("get_godot_version");
+ *   GDExtensionInterfaceGetGodotVersion get_godot_version = (GDExtensionInterfaceGetGodotVersion)(void*)p_get_proc_address("get_godot_version");
```

Comment:
```suggestion
 *   GDExtensionInterfaceGetGodotVersion get_godot_version = (GDExtensionInterfaceGetGodotVersion)(void *)p_get_proc_address("get_godot_version");
```
Should probably still follow the code style guidelines.